src/components/bs5/directionLinks/directionLinks.hbs

```diff
@@ -0,0 +1,9 @@
+{{! valid class values are: up, down, left, right}}
+<a class="qld-dir-link  {{~#if class}} {{class}}{{/if}}" 
+    {{~#if id}}id="{{id}}"{{/if~}} 
+    href="{{href}}" 
```

Comment:
Even though this is mandatory with the ``
~}}
`` on the closing of above. You won't have a space on the href tag, for new line split support also wrap this in a ``
{{~#if href}}
``